lib/mayaUsd/fileio/primUpdater.h

```diff
@@ -55,10 +55,19 @@ class UsdMayaPrimUpdater
         All = Push | Pull | Clear
     };
 
+    // pushCopySpecs result code.  Prune means success, but no further
+    // traversal should take place.
+    enum class PushCopySpecs
```

Comment:
It feels weird to have a enum have the same name as a function. Maybe PushCopyResult would have been a better name? In any case, this is not a widely used type, so it is not very important.